CryptoMarketCap/CryptoMarketCap/Scenes/Common/ChartView/ChartViewModel.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ChartViewModel.swift
+//  CryptoMarketCap
+//
+//  Created by Marek Slávik on 13.04.2021.
+//
+
+import Foundation
+import SwiftUI
+
+struct ChartViewModel: Identifiable {
+    struct Value {
+        let date: Date
+        let value: Double
+    }
+
+    let id = UUID()
+    let values: [Value]
+    let minimumValue: Double
+    let maximumValue: Double
+
+    init?(values: [Value], minimumValue: Double, maximumValue: Double) {
+        if values.isEmpty {
+            return nil
+        }
+
+        self.values = values
+        self.minimumValue = minimumValue
+        self.maximumValue = maximumValue
+        self.difference = maximumValue - minimumValue
+        self.lastDate = values.last!.date
```

Comment:
Now that we can return `nil`, I'd guard for the `values.last` so we can avoid force unwrapping.